src/index.ts

```diff
@@ -40,4 +44,12 @@ window.addEventListener('DOMContentLoaded', async () => {
       hideLoadingSpinner();
     }
   }
-});
+};
+
+if (isTestingEnvironment) {
```

Comment:
Jak dobrze rozumiem, to obecnie zakładacie, że dodanie skryptu przez `<script>` (albo w bezpośrednio kodzie strony) oznacza uruchomienie "produkcyjne", a dodanie skryptu po czasie (wklejenie do konsoli, lub osadzenie tego skryptu przez inny skrypt) oznacza uruchomienie "testowe".
Jednak to powinno **zawsze** móc działać produkcyjnie - ten skrypt będzie czasami osadzany przez naszą bibliotekę (bez konfiguracji przez App Store Shopera)